kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobActionDropdown.js

```diff
@@ -0,0 +1,202 @@
+define(['bluebird', 'common/events', 'common/html'], (Promise, Events, html) => {
```

Comment:
most of this is taken directly from the jobStateList module; there are a few minor modifications